src/gpl/src/replace.cpp

```diff
@@ -296,6 +297,29 @@ void Replace::doInitialPlace()
   ip_->doBicgstabPlace();
 }
 
+void Replace::runMBFF(int max_sz, float alpha, float beta, int threads) {
+
+  int num_flops = 0, num_paths = 0;
+  vector<float> x, y; 
+  vector<pair<int, int> > paths;
+
+  auto block = db_->getChip()->getBlock();
+  for (auto inst : block->getInsts()) {
+    auto mstr = inst->getMaster();
+    if (mstr->isSequential()) {
+      int x_i, y_i; 
+      inst->getOrigin(x_i, y_i);
+      x.push_back(x_i);
+      y.push_back(y_i);
```

Comment:
Does it make more sense to have a single vector of locations rather than 2 vectors of x and y?

Reply:
I don't think so... at that point it would be better to have vector<odb::Point>

Reply:
IMO the only reason to have a separate x and y vector is if x and y were accessed separately. To me it appears that x and y are always accessed together, so it should be a vector of `odb::Point`. It's more concise and probably gets better locality.